.gitignore

```diff
@@ -0,0 +1,68 @@
+#Maven
+
+target/
```

Comment:
Der `target`-Folder wird zwar ausgeschlossen, aber du hast dennoch ein paar Dateien darin in diesem PR. Die müssen wieder raus.